examples/SCAMPy/experiments/scm_pycles_pipeline/calibrate.jl

```diff
@@ -115,12 +128,15 @@ println("NUMBER OF ITERATIONS: ", N_iter)
 
 initial_params = construct_initial_ensemble(priors, N_ens, rng_seed=rand(1:1000))
 ekobj = EnsembleKalmanProcess(initial_params, yt, Γy, algo )
-scm_dir = "SCAMPy/"
+scampy_dir = "SCAMPy/"
 
 # Define caller function
-@everywhere g_(x::Array{Float64,1}) = run_SCAMPy(x, $param_names,
-   $y_names, $scm_dir, $ti, $tf, P_pca_list = $P_pca_list,
-   norm_var_list = $pool_var_list, scampy_handler = "call_BOMEX.sh") 
+@everywhere g_(x::Array{Float64,1}) = run_SCAMPy(
+    x, $param_names, $y_names, $scampy_dir, 
+    $scm_data_root, $scm_names, $t_starts, $t_ends,
+    P_pca_list = $P_pca_list, norm_var_list = $pool_var_list,
+    scampy_handler = "call_BOMEX.sh"
```

Comment:
can we build scampy_handler from "call_" + scm_name + ".nc" and have a 'call_CASENAME.sh' for each case?